constrained-voucher.mkd

```diff
@@ -155,12 +155,20 @@ The term Registrar Voucher Request, or acronym RVR, is introduced to refer to th
 
 {{RFC8366}} provides for vouchers that assert proximity, that authenticate the Registrar and that can offer varying levels of anti-replay protection.
 
+The proximity proof provided for in {{RFC8366}}, is an assertion that the Pledge and the Registrar are believed to be in close together, from a network topology point of view.
+Like in {{RFC8955}}, proximity shown by making TLS connections between the Pledge and Registrar using IPv6 Link-Local addresses.
```

Comment:
TLS -> DTLS in our case
link-local -> that's not the case; the Join-Proxy is typically used which can relay the DTLS records to a destination an arbitrary number of hops away.